src/licensedcode/models.py

```diff
@@ -737,6 +758,17 @@ def validate(licenses, verbose=False, no_dupe_urls=False):
         return errors, warnings, infos
 
 
+def get_yaml_safe_text(text):
+
```

Comment:
Add some docstring that explains what this does and why, as this is a key item IMHO

Reply:
Added, thanks!